docs/configure.md

```diff
@@ -49,29 +49,60 @@ The hubs are configured and deployed using *hub templates*. Because each hub
 type can be described by a template, with its own deployment chart, a hierarchy
 of hub types can be built and this makes development and usage easier.
 
-Currently there are two hub templates available:
-- the `base-hub` template
-- the `ephemeral-hub` template
+The graphic below, shows the relationship between the hub templates and the other
+config files and how they are merged together when deploying a pilot hub.
 
-For example, the **ephemeral hub** is a special kind of hub that is built using the *ephemeral-hub template*
-and has the following features:
+```{figure} images/config-flow.png
```

Comment:
@GeorgianaElena can you add a comment in these docs (with `% `) that says where these images were created, and how a different team member could update them if they wanted to do so?